EmployeeLib/Employee.cs

```diff
@@ -1,24 +1,25 @@
-﻿namespace EmployeeLib
+﻿using EmployeeLib.LSPModel;
+using System;
+
+namespace EmployeeLib
 {
-    public class Employee
+    public class Employee : IEmployee, ISalaryCalculator
     {
         public string FirstName { get; set; }
         public string LastName { get; set; }
-        public Employee Manager { get; set; } = null;
         public decimal Salary { get; set; }
 
-        public virtual void AssignManager(Employee manager)
-        {
-            // Simulate doing other tasks here - otherwise, this should be
-            // a property set statement, not a method.
-            Manager = manager;
-        }
-
         public virtual void CalculatePerHourRate(int rank)
         {
             decimal baseAmount = 12.50M;
 
             Salary = baseAmount + (rank * 2);
         }
+
+        public void GeneratePerformanceReview()
```

Comment:
why did you move this from CEO, the employee should not be able to generate performance reviews for himself or others. only the CEO